cmd/plural/api.go

```diff
@@ -63,6 +63,7 @@ func (p *Plural) apiCommands() []cli.Command {
 }
 
 func (p *Plural) handleInstallations(c *cli.Context) error {
+	p.InitPluralClient()
```

Comment:
Why do we need to init the plural client everywhere now?

Reply:
because parameters and env variables are taken in the runtime but the plural client needs the profile file before. In this case, the client should be initialized during command execution when the parameters are fetched